cypress/support/commands/publication-commands.js

```diff
@@ -14,14 +14,12 @@ import publicationSelectors from '../../selectors/publication.selectors';
  * @name createPublication
  * @memberOf Cypress.Chainable#
  * @function
- * @param {number} publicationNumber The publication number, must be unique
  * @param {string} shortTitle The short title for the case
  * @param {string} longTitle The long title for the case
  * @returns {Promise<String>} the id of the created publication-flow
  */
-function createPublication(publicationNumber, shortTitle, longTitle) {
+function createPublication(shortTitle, longTitle) {
```

Comment:
Ik snap waarom je dit zo hebt aangepast, je had hier ook de publicatienummer optioneel kunnen maken en die block wrappen met een if.
Dan kon je nog steeds een specifiek nummer meegeven als je wou.
Maar we passen dat wel aan als we dat nodig zouden hebben